src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -29,33 +29,21 @@
  */
 class AddConsoleCommandPass implements CompilerPassInterface
 {
-    private $commandLoaderServiceId;
-    private $commandTag;
-    private $noPreloadTag;
-    private $privateTagName;
-
-    public function __construct(string $commandLoaderServiceId = 'console.command_loader', string $commandTag = 'console.command', string $noPreloadTag = 'container.no_preload', string $privateTagName = 'container.private')
-    {
-        if (0 < \func_num_args()) {
-            trigger_deprecation('symfony/console', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
-        }
-
-        $this->commandLoaderServiceId = $commandLoaderServiceId;
-        $this->commandTag = $commandTag;
-        $this->noPreloadTag = $noPreloadTag;
-        $this->privateTagName = $privateTagName;
-    }
+    private const COMMAND_LOADER_SERVICE_ID = 'console.command_loader';
+    private const COMMAND_TAG = 'console.command';
+    private const NO_PRELOAD_TAG = 'container.no_preload';
+    private const PRIVATE_TAG_NAME = 'container.private';
```

Comment:
Please inline the strings. https://github.com/symfony/symfony/pull/40468#issuecomment-845791510